### PR TITLE
Adds the finnish quirk which gives you the spurdo mask, the alternative_screams var for items giving you different screams, and the wrestling kit for syndies.

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
read the title
oh yeah uhhh the spurdo screams are from Pyko, the hippie webmaster and the sprites too.

## Changelog
dreamseeker_0eJXmJwFqF

:cl:
add: WRESTLEMANIA TAKES OVER NANOTRASEN HIGHERUPS, STOCKS PLUMMET!
add: Finnish artefacts dug up from the nuclear hellzones of neo-Finland.
/:cl:

### CODE REVIEW

File: code/datums/quirks/neutral.dm [hunk 1]
```diff
@@ -299,6 +299,18 @@
 		)
 	)
 
+/datum/quirk/item_quirk/finnish
+	name = "Finland Enthusiast"
+	desc = "You have a powerful relic of old Finland in your posession."
+	value = 0
+	mob_trait = TRAIT_FINNABOO
+	gain_text = "<span class='notice'>You know enjoy everything about Finland.</span>"
```
Comment (AcapitalA) [inline code/datums/quirks/neutral.dm:307]: "know enjoy"
grammar [CENSORED FOR GIT] mode activated

Comment (karmaisblackandbluepilled) [reply code/datums/quirks/neutral.dm:307]: bazinga

**Review by TheChosenEvilOne — COMMENTED** [review]
just took a look at the word replacement, anyways what the fuck is with the indentation in that file

**Comment by karmaisblackandbluepilled** [comment]
> just took a look at the word replacement, anyways what the fuck is with the indentation in that file

I don't know

**Comment by karmaisblackandbluepilled** [comment]
merg

**Pull request merged**